decoder/semantic_tokens.go

```diff
@@ -606,3 +601,33 @@ func tokensForTupleConsExpr(expr *hclsyntax.TupleConsExpr, exprType cty.Type) []
 
 	return tokens
 }
+
+func tokensForTraversalExpression(traversal hcl.Traversal) []lang.SemanticToken {
```

Comment:
Nitpick: Since we're operating in the `decoder` package namespace, I think we should make this a little more specific, e.g. `semanticTokensForTraversalExpression`?

Reply:
Not a nitpick, its defensive programming 😁